src/assets/i18n/fr.json5

```diff
@@ -4016,6 +4019,32 @@
 
   // "process.overview.new": "New",
   "process.overview.new": "Nouveau",
+  
+   // "process.overview.table.actions": "Actions",
+   "process.overview.table.actions": "Actions",
+  
+  // "process.overview.delete": "Delete {{count}} processes",
+  "process.overview.delete": "Supprimer {{count}} processus",
+
+  // "process.overview.delete.processing": "{{count}} process(es) are being deleted. Please wait for the deletion to fully complete. Note that this can take a while.",
+  "process.overview.delete.processing": "{{count}} processus sont en train d'être supprimer. Patientez jusqu'à ce que la suppression soit terminée. Notez que cela peut prendre un certain temps.",
+
+  // "process.overview.delete.body": "Are you sure you want to delete {{count}} process(es)?",
+  "process.overview.delete.body": "Êtes vous sûr de vouloir supprimer {{count}} processus?",
+
+  // "process.overview.delete.header": "Delete processes",
+  "process.overview.delete.header": "Supprimer les processus",
+
+  // "process.bulk.delete.error.head": "Error on deleteing process",
+  "process.bulk.delete.error.head": "Erreur lors de la suppression de processus",
+
+  // "process.bulk.delete.error.body": "The process with ID {{processId}} could not be deleted. The remaining processes will continue being deleted. ",
+  "process.bulk.delete.error.body": "Le processus numéro {{processId}} n'a pu être supprimé. Les processus restants continuent à être supprimés. ",
```

Comment:
For the second phrase, I would rather go for "Les processus restants continueront à être supprimés."

Reply:
Done, added to new commit